docs/core/signing-transactions.md

```diff
@@ -0,0 +1,12 @@
+---
+id: spending-info
+title: Signing Transactions
+---
+
+# Signing Transactions in Bitcoin-S
+
+There are two kinds of transaction signing in Bitcoin-S, as defined in `Signer.scala`: Single Signing and Full Signing.
+
+Single signing, which is implemented by the subtypes of `SingleSigner`, consists of using a single `ECPrivateKey` (or other [Sign implementation](sign.md)) to generate a `PartialSignature` of the given transaction. This consists of an `ECDigitalSignature` and a `ECPublicKey`. This is useful any time you have a single private key and wish to get a single `ECDigitalSignature` for that private key as is the case when using [Partially Signed Bitcoin Transactions](psbts.md). In order to call `signSingle` on any instance of `SingleSigner`, you must have access to the (not signed) transaction being signed, as well as a `BitcoinUTXOSpendingInfoSingle`, which encapsulates all the information needed to sign with a single key.
```

Comment:
> you must have access to the (not signed) transaction being signed

this sounds weird, can just do "you must have access to the unsigned transaction"